Dockerfile

```diff
@@ -1,29 +1,68 @@
-FROM nginx:stable-alpine as ngx-php
-
-COPY config /build/config
-COPY src /build/src
-COPY third_party /build/third_party
-
+# Nginx Docker Playground (https://github.com/nginx-with-docker/nginx-docker-playground)
+FROM soulteary/prebuilt-nginx-modules:base-1.23.1-alpine AS Builder
```

Comment:
Please, try to use official docker images.

Reply:
@soulteary I think joanhey is right. haha

Reply:
I agree with this change, and will update this PR once I've finished a piece of content with a better scene, hopefully within the last two months of this year.

eg: https://www.zhihu.com/question/21483073/answer/2647942304

@rryqszq4 @joanhey